src/components/feedback/LinearProgress/LinearProgress.tsx

```diff
@@ -1,28 +1,37 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 import PropTypes from 'prop-types'
 import Box from '@mui/material/Box'
 import { LinearProgress as MuiLinearProgress, Label } from './LinearProgressStyles'
 import { isNil } from 'ramda'
-import { LinearProgressProps, ProgressColor } from './types'
+import { LinearProgressProps } from './types'
```

Comment:
If you removed the need for `ProgressColor`, we shouldn't export it anymore from './types'